account_payment_notification/i18n/es.po

```diff
@@ -0,0 +1,262 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* account_payment_notification
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-10-03 07:46+0000\n"
+"PO-Revision-Date: 2022-10-03 08:58+0100\n"
+"Last-Translator: Jairo Llopis <jairo@moduon.team>\n"
+"Language-Team: \n"
+"Language: es_ES\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.1.1\n"
+
+#. module: account_payment_notification
+#: model:mail.template,body_html:account_payment_notification.mail_template_notification
+msgid ""
+"<p>Dear <t t-out=\"object.partner_id.name or ''\">Azure Interior</t>,</p>\n"
+"            <p t-if=\"object.payment_type == 'inbound'\">\n"
+"                Your debit order was processed and sent to the bank.\n"
+"            </p>\n"
+"            <p t-else=\"\">\n"
+"                Your payment order was processed and sent to the bank.\n"
+"            </p>\n"
+"            <p t-if=\"object.reconciled_invoices_type == 'invoice'\">\n"
+"                It includes the following invoices:\n"
+"            </p>\n"
+"            <p t-else=\"\">\n"
+"                It includes the following credit notes:\n"
+"            </p>\n"
+"            <ul>\n"
+"                <t t-foreach=\"object.reconciled_invoice_ids | object."
+"reconciled_bill_ids\" t-as=\"invoice\">\n"
+"                    <li t-out=\"invoice.ref or invoice.name"
+"\">INV/2022/00001</li>\n"
+"                </t>\n"
+"            </ul>\n"
+"            <p>Total amount: <strong t-field=\"object.amount\"/></p>\n"
+"            <div>\n"
+"                <p>Thanks for your cooperation!</p>\n"
+"                <p>--<br/><t t-out=\"object.company_id.name\"/></p>\n"
+"                <p>\n"
+"                    <small>This is an automated notification. Replies will "
+"not be monitored nor answered. If you have questions, please contact us by "
+"the usual means.</small>\n"
+"                </p>\n"
+"            </div>\n"
+"        "
+msgstr ""
+"<p>Estimado/a <t t-out=\"object.partner_id.name or ''\">Azure Interior</t>,</"
+"p>\n"
+"            <p t-if=\"object.payment_type == 'inbound'\">\n"
+"                Su orden de débito ha sido procesada y enviada al banco.\n"
```

Comment:
@yajo in general in translation "débito" is "cobro" so we should change translation of "debit" with "cobro"

Please review all term and in spanish change "débito" to "cobro"

Thanks! 😄 

Great work!